examples/e-commerce/src/app.mobile.css

```diff
@@ -159,6 +159,15 @@
     max-width: 90%;
   }
 
+  .ais-SearchBox .ais-SearchBox-input,
+  .ais-RefinementList .ais-SearchBox-input {
+    font-size: 1rem;
```

Comment:
This makes the inputs font size at `16px`, which is the fix for mobile browsers on iOS to not zoom on the inputs.